config/jobs/containerd/containerd/containerd-postsubmit-jobs.yaml

```diff
@@ -0,0 +1,130 @@
+# These CI/post-submit jobs are used by Kubernetes tests to fetch a recent build of containerd instead of building containerd on every run.
+postsubmits:
+  containerd/containerd:
+    - name: post-containerd-build
+      labels:
+        preset-service-account: "true"
+      cluster: k8s-infra-prow-build
+      decorate: true
+      branches:
+        - main
+      spec:
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230616-e730b60769-master
+            command:
+              - runner.sh
+            args:
+              - test/build.sh
+            env:
+              - name: DEPLOY_DIR
+                value: main
+              - name: DEPLOY_BUCKET
+                value: k8s-staging-cri-tools
+            resources:
+              limits:
+                cpu: 4
+                memory: 6Gi
+              requests:
+                cpu: 4
+                memory: 6Gi
+      annotations:
+        testgrid-dashboards: sig-node-containerd,containerd-postsubmits
+        testgrid-tab-name: post-containerd-build
+        description: "builds development in progress branch of upstream containerd"
+
+    - name: post-containerd-build-1-6
+      labels:
+        preset-service-account: "true"
+      cluster: k8s-infra-prow-build
+      decorate: true
+      branches:
+        - release/1.6
+      spec:
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230616-e730b60769-master
+            command:
+              - runner.sh
+            args:
+              - test/build.sh
+            env:
+              - name: DEPLOY_DIR
+                value: release-1.6
+              - name: DEPLOY_BUCKET
+                value: k8s-staging-cri-tools
+            resources:
+              limits:
+                cpu: 4
+                memory: 6Gi
+              requests:
+                cpu: 4
+                memory: 6Gi
+      annotations:
+        testgrid-dashboards: sig-node-containerd,containerd-postsubmits
+        testgrid-tab-name: post-containerd-build-1.6
+        description: "builds release/1.6 branch of upstream containerd"
+
+    - name: post-containerd-build-1-7
```

Comment:
question: Where do the logs get uploaded to? In the older jobs there is a [`--upload=gs://kubernetes-jenkins/logs`](https://github.com/kubernetes/test-infra/blob/423fd8bb893e8d0576964ce309480fd504021b32/config/jobs/kubernetes/sig-node/containerd.yaml#L82)